node/rpc/src/wallet_api.rs

```diff
@@ -234,34 +230,6 @@ where
     Ok(SignatureJson(sig))
 }
 
-/// Sign an `UnsignedMessage`, return `SignedMessage`
```

Comment:
I am not sure about removing this one. Seems like it can be easily covered and is quite useful if we move to a standalone wallet binary.

Reply:
It can be implemented using the `sign` RPC, can't it?

Reply:
I think so, yeah.

Reply:
Okay, let's just keep the `sign` RPC call, then. If `sign_message` becomes convenient in the future, we can restore this code.